tests/base_test.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
```

Comment:
Remove the shebang, we shouldn't include it for any new files unless it's needed.